query/src/main/java/org/infinispan/query/impl/InfinispanQueryStatisticsInfo.java

```diff
@@ -17,13 +17,6 @@
 public class InfinispanQueryStatisticsInfo implements InfinispanQueryStatisticsInfoMBean {
 
```

Comment:
There is another workaround here: https://github.com/Sanne/infinispan/blob/UpgradeSearchComponents/query/src/main/java/org/infinispan/query/backend/SearchFactoryHandler.java#L88-90

Do we still need it?